environment.yml

```diff
@@ -3,58 +3,58 @@ channels:
 - defaults
 - conda-forge
 dependencies:
-- awscli=1.15.4
-- beautifulsoup4=4.5.3
-- botocore=1.10.4
-- boto=2.46.1
-- boto3=1.7.4
-- bqplot=0.10.2
-- cython=0.27.3
-- dask=0.17.2
+- awscli=1.16.96
+- beautifulsoup4=4.7.1
+- botocore=1.12.86
+- boto=2.49.0
+- boto3=1.9.86
+- bqplot=0.11.5
+- cython=0.29.4
+- dask=0.20.2
 - feather-format=0.4.0
-- ipython=6.1.0
-- ipywidgets=7.1.0
-- jinja2=2.9.6
-- jsonschema=2.5.1
+- ipython=6.5.0
+- ipywidgets=7.4.2
+- jinja2=2.10
+- jsonschema=2.6.0
 - jupyter=1.0.0
 - libffi=3.2.1
 - libgfortran=3.0.0
-- libtiff=4.0.6
-- libxml2=2.9.2
-- matplotlib=2.2.2
+- libtiff=4.0.10
+- libxml2=2.9.8
+- matplotlib=2.2.3
 - nomkl=1.0
-- notebook=5.4.1
+- notebook=5.7.4
 - nose=1.3.7
-- numexpr=2.6.2
-- numpy=1.13.3
-- openblas=0.2.20
-- pandas=0.22.0
-- patsy=0.4.1
-- psycopg2=2.6.2
-- pyarrow=0.8.0
+- numexpr=2.6.9
+- numpy=1.14.6
+- openblas=0.3.3
+- pandas=0.24.1
+- patsy=0.5.1
+- psycopg2=2.7.7
+- pyarrow=0.12.0
 - pycrypto=2.6.1
-- pytest=3.1.3
-- python=3.6.4
-- pyyaml=3.12
-- requests=2.18.4
-- s3fs=0.1.2
-- seaborn=0.8
-- scipy=1.0.1
-- scikit-learn=0.19.1
-- statsmodels=0.8.0
-- urllib3=1.22
-- xgboost=0.6a2
+- pytest=3.10.1
+- python=3.7.1
+- pyyaml=3.13
+- requests=2.21.0
+- s3fs=0.2.0
+- seaborn=0.9.0
+- scipy=1.2.0
+- scikit-learn=0.20.2
+- statsmodels=0.9.0
+- urllib3=1.24.1
+- xgboost=0.81
 - pip:
-  - civis==1.9.0
-  - civisml-extensions==0.1.8
-  - cloudpickle==0.5.2
-  - dropbox==7.1.1
+  - civis==1.9.3
+  - civisml-extensions==0.1.10
+  - cloudpickle==0.7.0
```

Comment:
There's a `cloudpickle` v0.8.0 available on conda-forge, and conda is already installing it (I don't know where the dependency is). I suggest we move this from the pip to the conda section, and use v0.8.0.